lib/kb_uploadmethods/Utils/UnpackFileUtil.py

```diff
@@ -38,14 +39,30 @@ def _file_to_staging(self, file_path_list, subdir_folder=None):
         subdir_folder_str = '/' if not subdir_folder else '/{}'.format(subdir_folder)
         staging_service_host = self._staging_service_host()
         end_point = staging_service_host + '/upload'
-        headers = {'Authorization': self.token}
 
         files = {'destPath': subdir_folder_str}
 
         for file_path in file_path_list:
             files.update({'uploads': (os.path.basename(file_path), open(file_path, 'rb'))})
-
-            resp = _requests.post(end_point, headers=headers, files=files)
+            
+            try:
+                resp = _requests.post(end_point, 
```

Comment:
@Tianhao-Gu @dauglyon Are we sure this is the intended behavior? Will this try to upload the same files, then if one of them is over 2gb, then it will re-upload all of the files all over again? 
Maybe we should just write a simple unit test for this or just do a manual test?

Reply:
The error that is caught (Overflow) occurs before the actual request is made, it is the library throwing the input as "too big to process" basically

Reply:
I'm definitely open to testing strategies. The existing code is mocked in the tests.

Also, counter to the variable naming, files are uploaded one at a time, so a single file being "too big" would not cause the rest to retry, only that one

Reply:
Some test thoughts
1) I don't know that the maximum size is the same in all environments. It seems it might be dependent on the systems C installation
2) we would want to generate the large test file rather than bundle it into the repo (no one wants a 2gb file in a git repo)

Reply:
I am ok testing this in CI. I cannot think of a good way to test it with unit tests. I think the test docker container doesn't have access to the staging area.

Reply:
Sounds good to me. Let's get this registered on CI

Reply:
This worked for me on ci. I bumped the kbase.yml and updated release notes. Can we get this merged and registered on appdev @Tianhao-Gu  ?

Reply:
Sure. Thanks for testing!